dts/arm/nxp/nxp_mcxw7x_common.dtsi

```diff
@@ -434,9 +435,9 @@
 		reg = <0x2000 0x140>;
 		clocks = <&scg SCG_K4_EDMA_CLK 0x410>;
 		interrupts = <2 0>, <3 0>, <4 0>, <5 0>,
-			<6 0>, <7 0>, <8 0>, <9 0>,
-			<10 0>, <11 0>, <12 0>, <13 0>,
-			<14 0>, <15 0>, <16 0>, <17 0>;
+			     <6 0>, <7 0>, <8 0>, <9 0>,
```

Comment:
I think the number of tabs you should have should only be to align with the "interrupts" then the rest should be spaces. This will ensure the alignment you are trying to get no matter what someone's editor has for tabs.

this is repeated in other changes